app/etl/item/content/parsers/body_content.rb

```diff
@@ -1,6 +1,9 @@
 class Item::Content::Parsers::BodyContent
   def parse(json)
-    json.dig("details", "body")
```

Comment:
Before merging, I would rather finding why our integration tests are not covering this problem. I am currently working on this.